core/src/main/java/org/frankframework/align/content/JsonDocumentContainer.java

```diff
@@ -91,12 +98,80 @@ protected void addContent(JsonElementContainer parent, JsonElementContainer chil
 		parent.addContent(child);
 	}
 
+	public Message toMessage() throws IOException {
+		return toMessage(true);
+	}
+
+	public Message toMessage(boolean indent) throws IOException {
```

Comment:
Misschien mooi hier even bij te vermelden dat dit streaming is, of de tostring te deprecaten ofzo?